<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="filesList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="compButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>сравнить</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>751</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>папка 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fold1Label">
       <property name="minimumSize">
        <size>
         <width>561</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>561</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFloder1">
       <property name="text">
        <string>выбор</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>751</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>папка 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fold2Label">
       <property name="minimumSize">
        <size>
         <width>561</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>561</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFloder2">
       <property name="text">
        <string>выбор</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
